bpm/bonita-core/bonita-login/bonita-login-api-impl/src/main/java/org/bonitasoft/engine/core/login/SecuredLoginServiceImpl.java

```diff
@@ -112,7 +112,8 @@ public SSession login(final Map<String, Serializable> credentials) throws SLogin
         } catch (final AuthenticationException ae) {
             throw new SLoginException(ae);
         } catch (final SUserNotFoundException e) {
-            throw new SLoginException("Unable to found user " + userName);
+            warningLog("Unable to find user with username " + userName + " in database.");
+            throw new SLoginException("Unable to find user " + userName);
         } finally {
```

Comment:
Or we could throw an exception that extend LoginException (like UserNotFoundOnLoginException)